tests/foreman/api/test_contentviewversion.py

```diff
@@ -310,6 +320,595 @@ def test_negative_delete(self):
         # Make sure that content view version is still present
         self.assertEqual(len(content_view.read().version), 1)
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_remove_renamed_cv_version_from_default_env(self):
+        """Remove version of renamed content view from Library environment
+
+        @id: 7d5961d0-6a9a-4610-979e-cbc4ddbc50ca
+
+        @Steps:
+
+        1. Create a content view
+        2. Add a yum repo to the content view
+        3. Publish the content view
+        4. Rename the content view
+        5. remove the published version from Library environment
```

Comment:
You start the last step with lower case in several docstrings, sorry for the nitpick but the docs person in me cannot unsee it :)